Week1/MAKEME.md

```diff
@@ -53,7 +55,7 @@ Write a JavaScript file (to be executed with Node.js) that queries (using select
 9. What's the top 10 most populated cities?
 10. What is the population number of the world?
 
-After you've written your queries, test to see if they work by executing `node <FILE_NAME>`.
+After you've written your queries, and tested them, see if the file work by executing `node <FILE_NAME>`.
```

Comment:
I think they need to execute with `node filename` in order to test queries because they  are writing the queries in the JS file.